test/optimus/pr/ValidateOptimus.wdl

```diff
@@ -58,14 +69,14 @@ task ValidateBam {
 
         if [ "$calculated_checksum" == ${expected_checksum} ]
         then
-             echo Computed and expected bam hashes match \( $calculated_checksum \)
+             echo Computed and expected bam hashes match \( "$calculated_checksum" \)
              printf PASS > result.txt
         else 
-             echo Computed \( $calculated_checksum \) and expected \( ${expected_checksum} \) bam file hashes do not match
+             echo Computed \( "$calculated_checksum" \) and expected \( ${expected_checksum} \) bam file hashes do not match
```

Comment:
why is `calculated_checksum` quoted but `expected_checksum` not?

Reply:
done